packages/vaex-core/vaex/dataframe_protocol.py

```diff
@@ -315,12 +323,17 @@ def __init__(self, column: vaex.expression.Expression, allow_copy: bool = True)
         self._col = column
         self._allow_copy = allow_copy
 
-    @property
     def size(self) -> int:
         """
         Size of the column, in elements.
+
+        Corresponds to DataFrame.num_rows() if column is a single chunk;
+        equal to size of this current chunk otherwise.
+
+        Is a method rather than a property because it may cause a (potentially
+        expensive) computation for some dataframe implementations.
         """
-        return self._col.df.count("*")
+        return int(self._col.df.count("*"))
```

Comment:
Just a quick change while we change this line
```suggestion
        return int(len(self._col.df))
```

Btw, just noticed that pandas doesn't have this as a property:
https://github.com/pandas-dev/pandas/blob/f0814bcca41a116794ff926b571ec6c2d15685d7/pandas/core/interchange/column.py#L84

Reply:
Ah yeah, wrote an issue https://github.com/pandas-dev/pandas/issues/48392